tests/test_utils.cpp

```diff
@@ -417,3 +416,19 @@ void compareReset(const libcellml::ResetPtr &r1, const libcellml::ResetPtr &r2)
     EXPECT_EQ(r1->testValueId(), r2->testValueId());
     EXPECT_EQ(r1->resetValueId(), r2->resetValueId());
 }
+
+void expectEqualFileContents(const std::string &fileName, const std::string &fileContents)
+{
+    // Uncomment the below when you want to generate the expected file contents.
+    // #define NEW_GENERATOR
```

Comment:
This sort of thing should be done with a CMake command so you can run a target that updates all the file contents.